ecosystem/sep-0006.md

```diff
@@ -311,15 +311,26 @@ Name | Type | Description
 -----|------|------------
 `type` | string | Always set to `non_interactive_customer_info_needed`.
 `fields` | list of strings | A list of field names that need to be transmitted via SEP-12 for the deposit to proceed.
+`sep12_type` | string | (optional) The SEP-12 `type` value associated with `fields`. Wallets may use this value in [SEP-12 GET /customer](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#customer-get) requests for more information on the returned list.
```

Comment:
💡 Is `fields` essentially deprecated with the addition of type? If there's redundant data here I think we should say the fields field is deprecated. It should still be provided but new applications should use the type. Is that accurate?

Reply:
Thats accurate, I'll deprecate the field.